.travis.yml

```diff
@@ -1,10 +1,7 @@
 sudo: false
 language: python
 python:
- - "2.7"
- - "3.4"
- - "3.5"
- - "3.6"
+ - "3.7"
```

Comment:
Why removing the old py3 version? Are we making any changes here that bump the minimum version of python required? (Cf also the minimum version set in the README below).

Reply:
@pypingou Thanks for the feedback. Using python 3.7 as per suggestion by CVerna.